lib/install/config/assets.js

```diff
@@ -0,0 +1,40 @@
+// Handles static asset integration - like sass and images
+// Note: You must restart bin/webpack-watcher for changes to take effect
+
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const process = require('process')
+const sharedConfig = require('./shared.js')
+const { devServer } = require('../../package.json')
+
+const production = process.env.NODE_ENV === 'production'
+const hotServerAddr = `http://${devServer.host}:${devServer.port}/`
```

Comment:
That's correct, but again like above. I can't use same host and port. Say, I configured dev server to run on port - 4000, then I have to change that here too. This is what I was referring when I said it was complicated to read this config out of ruby.  Packs directory seems like a core config and not sure if environment variable `WEBPACK_DIST_DIR` is right place to store/override it. 

Obviously, I understand that this added more code, but things are more configurable and is at one place going forward.